e2e/tests/functional/plugins/displayLayout/displayLayout.e2e.spec.js

```diff
@@ -427,6 +427,63 @@ test.describe('Display Layout', () => {
     // In real time mode, we don't fetch annotations at all
     expect(networkRequests.length).toBe(0);
   });
+
+  test('We can preview plot in display layouts', async ({ page, openmctConfig }) => {
```

Comment:
Let's move this out of the displayLayout.e2e.spec.js file and into another suite. This suite is too large with playwright's sharding method

Reply:
done